doc/whats-new.rst

```diff
@@ -26,7 +26,8 @@ New Features
 
 Breaking changes
 ~~~~~~~~~~~~~~~~
-
+- adjust the deprecation policy for python to once again align with NEP-29 (:issue:`7765`, :pull:`7793`)
```

Comment:
Not really a breaking change, maybe an internal change?

Reply:
a bit more visible than an internal change, this has implications for users. But yes, I agree "breaking changes" might not be the right category. However, that's what we've been using in the past for policy changes: https://docs.xarray.dev/en/stable/whats-new.html#id308